beacon-chain/state/state-native/getters_epbs.go

```diff
@@ -31,6 +32,10 @@ func (b *BeaconState) IsParentBlockFull() (bool, error) {
 	b.lock.RLock()
 	defer b.lock.RUnlock()
 
+	if b.slot == primitives.Slot(params.BeaconConfig().EPBSForkEpoch)*params.BeaconConfig().SlotsPerEpoch {
+		return true, nil
```

Comment:
first slot of EIP7732 should always be full

Reply:
This will also overflow on any configuration that has maxuint for epbs fork epoch.